docs/pages/resources.md

```diff
@@ -42,8 +42,8 @@ Use a SaaS service as a backend for functionality on your Jekyll site
 ### Content Management
   - [CloudCannon](https://cloudcannon.com/): The Cloud CMS for Jekyll
   - [Contentful](https://github.com/contentful/jekyll-contentful-data-import): Content infrastructure for digital teams
-  - [Forestry.io](https://forestry.io/): A free Git-based responsive CMS, with content modeling and instant previews.
-  - [Netlify CMS](https://www.netlifycms.org/): Open source content management for your Git workflow
+  - [TinaCMS.io](https://tina.io/): Tina is an open-source, Git-backed headless content management system (CMS) that empowers both developers and content creators to collaborate seamlessly on a single platform
+  - [Decap CMS](https://decapcms.org/): Decap CMS (formerly Netlify CMS) is an open source content management system for your Git workflow that enables you to provide editors with a friendly UI and intuitive workflows
```

Comment:
Hi @mattr 👋 

There is a [spell check that fails](https://github.com/jekyll/jekyll/actions/runs/6287586582/job/17072199923#step:2:506)  and it's coming from this PR

```
Error: docs/pages/resources.md:45:26 ... 30, Error - `tina` is not a recognized word. (unrecognized-spelling)
Error: docs/pages/resources.md:46:25 ... 33, Error - `decapcms` is not a recognized word. (unrecognized-spelling)
```

I wonder if you happen to know how can we fix it.
Thanks in advance.

Reply:
Hi @JuanVqz. I think you want @mattr- (don't worry, it happens all the time 😆 )